src/app/_styles/base.css

```
@@ -104,3 +104,27 @@ input {
 .fixed-overlay-safe {
   padding: var(--safe-top) var(--safe-right) var(--safe-bottom) var(--safe-left);
 }
+
+/* Safe area 변수 (현대 iOS: env, 구형 iOS: constant) */
+:root {
+  --safe-top: env(safe-area-inset-top);
+  --safe-bottom: env(safe-area-inset-bottom);
+}
+@supports (padding: constant(safe-area-inset-top)) {
+  :root {
+    --safe-top: constant(safe-area-inset-top);
```

Biome (2.1.2)

[error] Unexpected unknown function: constant

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)